src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -1084,12 +1084,8 @@ private ExtRequest prepareExt(String bidder,
                 ? mapper.mapper().valueToTree(ExtPrebidBidders.of(prebidParameters))
                 : null;
 
-        final ExtRequestPrebid.ExtRequestPrebidBuilder extPrebidBuilder = extPrebid != null
-                ? extPrebid.toBuilder()
-                : ExtRequestPrebid.builder();
```

Comment:
Why have you removed this?

Reply:
Because this case never happen and it can be simplified like a Objects.requireNonNull()

Reply:
Objects.requireNonNull() throws NPE, in case of no present value, it's not suitable for this case